integrations/extensions/starter-kits/language-model-conversational-search/README.md

```diff
@@ -63,24 +65,117 @@ If this does not suit your needs and you want to experiment with different promp
 
 1. Go in the "Generate Answer" action and edit Step 5.
 
-1. Edit the value of the `prompt` session variable. 
+1. Edit the value of the `prompt` session variable.
 
-1. Modify the text of the prompt passed to the language model.  
+1. Modify the text of the prompt passed to the language model.
 
-Also, you can modify the `model_id` session variable to control which model is used.  You can see which models are available on [the model compatibility page of the watsonx tech preview API](https://bam.res.ibm.com/docs/models).
+Also, you can modify the `model_id` session variable to control which model is used. You can see which models are available on [the model compatibility page of the watsonx tech preview API](https://bam.res.ibm.com/docs/models).
```

Comment:
https://bam.res.ibm.com/docs/models this link seems to say that it should NOT be shared externally

Reply:
Good catch!  This should be https://workbench.res.ibm.com/docs/models .  I believe it is only visible to users who are part of the tech preview, so you will probably need to take my word for it.